Airports/EGCC/SMR/Geo.txt

```diff
@@ -98,185 +98,191 @@ N053.22.04.850 W002.16.59.239 N053.22.03.403 W002.17.00.289 smrCCstand
 N053.22.02.170 W002.16.58.887 N053.22.03.730 W002.16.57.783 smrCCstand
 ;213
 N053.22.03.052 W002.16.56.472 N053.22.01.418 W002.16.57.597 smrCCstand
-; - 212 (WFU AIRAC 1708)
+;212 (WFU AIRAC 1708)
 ;N053.22.00.077 W002.16.55.362 N053.22.01.528 W002.16.52.939 smrCCstand
-; - 211
+;211
```

Comment:
Since there is only one line of Geo related to these stands, the comment should be at the end of the line. So:

‘N053.[...] smrCCstand ;211’